app/pathfinder_gui.py

```diff
@@ -206,14 +214,13 @@ async def return_scanner_configuration(self, data):
         else:
             return dict(name=scanner, error='scanner not able to be found')
 
-    def load_scanners(self):
+    async def load_scanners(self):
```

Comment:
Outside scope for this PR, but not sure of runtime behavior of this file I/O here - that is even though we have the `async` tag on the func, none of the internal calls are awaited and we are not using the asyncio versions of these file libraries.

(just something to note for future)